### PR TITLE
add argo binding

### PR DESCRIPTION
**Pull request by rg2011**
Adds support for internal SOAP service

### CODE REVIEW

**Comment by rg2011** [comment]
> I'd suggest to add this entry to CHANGES_NEXT_RELEASE file:
> 
> ```
> - Add: ARGO binding
> ```

fixed

File: lib/bindings/ARGOBinding.js
Comment (rg2011) [reply lib/bindings/ARGOBinding.js]: renoved

File: lib/bindings/ARGOBinding.js
Comment (AlvaroVega) [inline lib/bindings/ARGOBinding.js:422]: Maybe some configuration (host, port, remote_url and a disable flag) could be added to https://github.com/telefonicaid/iotagent-json/blob/master/lib/configService.js

**Review by fgalan — APPROVED** [review]
LGTM with regards to my comments

@AlvaroVega has also done some comment, so the PR needs first a LGTM on his side before be merged.

File: lib/bindings/ARGOBinding.js
Comment (rg2011) [reply lib/bindings/ARGOBinding.js:422]: the host, port and disable flag do exist, but I kept them all inside this file to make the service self-contained and "pluggable". In the future you could deprecate this feature from the iota-json anytime (e.g. because it migrates to a dedicated iota), just by removing this file and its tests.

But if you feel it is better to move some of the constants, flags or errors to the shared files, I am ok with that too.

**Review by AlvaroVega — APPROVED** [review]
LGTM

File: lib/bindings/ARGOBinding.js
Comment (fgalan) [reply lib/bindings/ARGOBinding.js:422]: NTC at the end.

**Pull request merged**
